lib/console.js

```diff
@@ -249,6 +253,86 @@ Console.prototype.groupEnd = function groupEnd() {
     this[kGroupIndent].slice(0, this[kGroupIndent].length - 2);
 };
 
+const untablable = (value) =>
+  value == null ||
+  ['symbol', 'undefined', 'boolean', 'number', 'string']
```

Comment:
`undefined` is redundant to `value == null`. Either the check above should be strict equal or the `undefined` should be removed here.